HW_8_bulls.py

```diff
@@ -0,0 +1,61 @@
+import os
+os.system('cls')
+
+n = '6521'
+n_tup = tuple(str(n))
+
+
+def init():                                                  # ввод числа пользователем
+    num = input('Введите четырехзначное число ')
+    return num
+
+
+def is_num_valid(num):                   # проверка введенного числа на повторяющиеся цифры и длину
+    is_valid = ''
+    if len(num) != 4:
+        is_valid = 'length'
+    else:
+        for i in num:
+            if num.count(i) == 1:
+                is_valid = 'true'
+            elif num.count(i) > 1:
+                is_valid = 'false'
+                break
+    return is_valid
```

Comment:
You can improve verification and return bool value instead of string
```suggestion
def is_num_valid(num):                   # проверка введенного числа на повторяющиеся цифры и длину
    if not len(num) == 4 or not len(set(num)) == 4 ....
        return False
.....
```

Reply:
Done